apps/juicebox_web/assets/js/videos/video.scss

```diff
@@ -33,9 +33,20 @@
   bottom: 0;
   display: flex;
   width: 100%;
-  height: 50px;
-  padding: 0 50px;
+  min-height: 50px;
+  padding: 0 50px 0 0;
   font-size: 17px;
   background-color: #f2f2f2;
   align-items: center;
+  h3 {
```

Comment:
Rule declaration should be preceded by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)